database/src/main/resources/db/migration/V1.77.0__RHCLOUD-25998_add_drawer_notification_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE public.drawer_notification (
+    user_id varchar(50) NOT NULL,
+    org_id text NOT NULL,
+    event_id uuid NOT NULL,
+    created timestamp NOT NULL,
+    read boolean NOT NULL default false,
+    CONSTRAINT pk_drawer_notification PRIMARY KEY (user_id, event_id)
```

Comment:
This can be changed to a `unique` constraint. I would also include `org_id` in that constraint because all queries will include an `org_id` and Postgres will perform faster if that field is part of the `unique` constraint.

Reply:
Ok